src/map/script.cpp

```diff
@@ -61,6 +61,7 @@
 #include "pet.hpp"
 #include "quest.hpp"
 #include "storage.hpp"
+#include "tax.hpp" // e_tax_type
```

Comment:
```suggestion
```
Is this needed if we don't use any script commands for it?

Reply:
Yes because script_constants.hpp is using them.

Reply:
Very true, forgot that was also imported here! Ignore this suggestion!